sphinx/templates/quickstart/conf.py_t

```diff
@@ -1,119 +1,71 @@
 # Configuration file for the Sphinx documentation builder.
 #
-# This file only contains a selection of the most common options. For a full
-# list see the documentation:
+# For the full list of built-in configuration values, see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
+{% if append_syspath -%}
 # -- Path setup --------------------------------------------------------------
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-{% if append_syspath -%}
 import os
 import sys
 sys.path.insert(0, {{ module_path | repr }})
-{% else -%}
-# import os
-# import sys
-{% if module_path -%}
```

Comment:
Note: Both`append_syspath` and `module_path` are available on apidoc. And quickstart does not passes both of them. So this if-condition is not needed.